sqlglot/generator.py

```diff
@@ -389,7 +389,10 @@ def tablealias_sql(self, expression):
         return f"{alias}{columns}"
 
     def bitstring_sql(self, expression):
-        return f"b'{self.sql(expression, 'this')}'"
+        return f"{self.sql(expression, 'this')}"
```

Comment:
so the default just returns a normal string?

return self.sql(expression, "this")

Reply:
Stored as a decimal string, so yeah.